model/tensorflow_hub/examples/tfhub_text_classifier/test_textclassifier.py

```diff
@@ -0,0 +1,71 @@
+import os
+import ast
+import sys
+import json
+import tempfile
+import contextlib
+import subprocess
+import unittest.mock
+
+from dffml.model.model import ModelContext, Model, ModelNotTrained
+from dffml.util.os import chdir
+
+
+def sh_filepath(filename):
+    return os.path.join(os.path.dirname(__file__), filename)
+
+
+@contextlib.contextmanager
+def directory_with_csv_files():
+    with tempfile.TemporaryDirectory() as tempdir:
+        with chdir(tempdir):
+            subprocess.check_output(["bash", sh_filepath("train_data.sh")])
+            subprocess.check_output(["bash", sh_filepath("test_data.sh")])
+            yield tempdir
+
+
+class TestExample(unittest.TestCase):
+    def python_test(self, filename):
+
+        # Path to target file
+        filepath = os.path.join(os.path.dirname(__file__), filename)
+
+        # Capture output
+        stdout = subprocess.check_output([sys.executable, filepath])
+        lines = stdout.decode().split("\n")
+
+        # Check the Accuracy
+        self.assertEqual(lines[-3], "Accuracy: 0.6666666865348816")
```

Comment:
Let's not do an exact check on accuracy. Instead let's split this line and check the the first part is "Accuracy:" and then ext part is a float.

Reply:
Oh, alright.